### PR TITLE
Fedora 41 fixes

### PR DESCRIPTION
**Pull request by LukeShortCloud**

### CODE REVIEW

**Comment by LukeShortCloud** [comment]
Disabling the debug package fixes Fedora 41 for all our packages. For example: https://github.com/playtron-os/rpm-specs-gaming/pull/93#issuecomment-2359586997

Mesa is unaffected but I wanted to update it to (1) trigger an automatic rebuild in Fedora Copr and (2) get us to the latest patch release. This is inline with what Fedora 40 uses (Mesa 24.1.7). I did a test build on Fedora 41 and it worked. When we actually ship a public build with Fedora 41 (which is a more long-term goal), we can upgrade the minor version to Mesa 24.2.

**Comment by LukeShortCloud** [comment]
The `gamescope-session-playtron` CI pipeline fails to install the package once it's built. This is because it's missing a couple of dependencies.

```
Run dnf -y install ~/rpmbuild/RPMS/*/*.rpm
  dnf -y install ~/rpmbuild/RPMS/*/*.rpm
  shell: sh -e {0}
  env:
    PACKAGE_NAME: gamescope-session-playtron
Last metadata expiration check: 0:00:15 ago on Fri Sep 20 17:17:42 2024.
Error: 
 Problem: conflicting requests
  - nothing provides gamescope-session needed by gamescope-session-playtron-0.2.2-2.fc40.x86_64 from @commandline
  - nothing provides grid needed by gamescope-session-playtron-0.2.2-2.fc40.x86_64 from @commandline
(try to add '--skip-broken' to skip uninstallable packages)
```

https://github.com/playtron-os/rpm-specs-gaming/actions/runs/10963527663/job/30445190954?pr=94

I added a commit that installs two missing repositories that contain those dependencies.

File: mesa/mesa.spec [hunk 1]
```diff
@@ -77,11 +77,6 @@ Source0:        https://archive.mesa3d.org/mesa-%{ver}.tar.xz
 Source1:        Mesa-MLAA-License-Clarification-Email.txt
 
 Patch10:        gnome-shell-glthread-disable.patch
-Patch11:        0001-llvmpipe-Init-eglQueryDmaBufModifiersEXT-num_modifie.patch
-Patch12:        0001-Revert-ac-radeonsi-remove-has_syncobj-has_fence_to_h.patch
-
-# s390x only
-Patch100:       fix-egl-on-s390x.patch
```
Comment (alkazar) [inline mesa/mesa.spec:84]: Were these patches upstreamed? Or not needed anymore?

**Review by alkazar — APPROVED** [review]

**Comment by LukeShortCloud** [comment]
All packages have been uploaded and built for Fedora 41 now!

https://copr.fedorainfracloud.org/coprs/playtron/gaming/packages/

**Pull request merged**